polaris/ocean/ocean.cfg

```diff
@@ -0,0 +1,15 @@
+# This config file has default config options for the ocean component, which
+# can either be MPAS-Ocean or OMEGA
+
+# Options related the ocean component
+[ocean]
+# Which model, MPAS-Ocean or OMEGA, is used
+model = mpas-ocean
```

Comment:
@xylar Do you have in mind that users would manually change this entry to switch between models? 
And/or that there would be an argument to setup tests/suites that would be `--model`?

Reply:
Yes, both of those should be possible.  I think a `--model` flag will definitely be important for convenience of use (e.g. if you don't have any reason to have a config file).

We'll also have to think through how we indicate which test cases only work for one model or the other (how an exception is raised when the wrong model is requested).

Reply:
It's a little weird to have a system-level flag for a component-specific config option so maybe the section should not be ocean but instead be something more generic like `[component]`.

Reply:
I think it's important to have a `model` option for each component (ocean, land-ice, etc) so that there is a default model specified for each component and users could run setup for each component without specifying model via flags or changing the cfg files. I don't know if this is compatible with the generic heading you're suggesting.